yaml-tests/src/test/java/MultiServerJDBCYamlTests.java

```diff
@@ -69,6 +73,165 @@ public MultiServerInitialConnectionExternalTests() {
         }
     }
 
+    /**
+     * Concrete implementation of the test class that forces continuationa on every query.
+     */
+    @Nested
+    public static class MultiServerForceContinuationsTests extends MultiServerJDBCYamlTests {
+        public MultiServerForceContinuationsTests() {
+            super(1);
+        }
+
+        @Override
+        public Map<String, Object> getAdditionalOptions() {
+            return Map.of(OPTION_FORCE_CONTINUATIONS, true);
+        }
+
+        @Override
+        @Disabled("Infinite continuation loop")
```

Comment:
Instead of just comments, should these reference issues?

Reply:
Done.